File: e2e/selenide-junit5-selenium-grid/src/test/java/test/desktop/createTask/CreateTaskUatTest.java
package test.desktop.createTask;

import io.qameta.allure.Epic;
import io.qameta.allure.Feature;
import io.qameta.allure.TmsLink;
import test.base.DesktopTest;
import test.spec.createTask.ICreateTaskUatTest;

import static data.AllureEpic.TASK_MANAGEMENT;

@Epic(TASK_MANAGEMENT)
@Feature("Create task")
@TmsLink("100")
class CreateTaskUatTest extends DesktopTest implements ICreateTaskUatTest {}

File: demo-android/app/src/androidTest/java/com/example/demo/e2e/test/create/CreateTaskUatTest.kt
package com.example.demo.e2e.test.create

import com.example.demo.e2e.context.TaskTestContext
import com.example.demo.e2e.data.AllureEpic.TASK_MANAGEMENT
import com.example.demo.e2e.test.base.UatTestBase
import io.qameta.allure.kotlin.Epic
import io.qameta.allure.kotlin.Feature
import io.qameta.allure.kotlin.TmsLink
import org.junit.Test

@Epic(TASK_MANAGEMENT)
@Feature("Create task")
@TmsLink("100")
class CreateTaskUatTest : UatTestBase() {

    @Test
    fun shouldCreateTask() {
        // Given
        val context = TaskTestContext()
        steps.navigation.openMainPage()

        // When
        steps.tasks
            .openCreateTaskForm()
            .setTaskData(context.createTaskData())
            .submitForm()

        // Then
        validate.tasks.hasTask(context.title)

        // When
        steps.tasks.openTaskInfoForm(context.title)

        // Then
        validate.task.data(context.createTaskData())
    }
}
